firebase-config/src/main/java/com/google/firebase/remoteconfig/RemoteConfigComponent.java

```diff
@@ -209,11 +205,7 @@ private ConfigCacheClient getCacheClient(String namespace, String configStoreTyp
     return getCacheClient(context, appId, namespace, configStoreType);
   }
 
-  /**
-   * The {@link LegacyConfigsHandler} needs access to multiple cache clients, and the simplest way
-   * to provide it access is to keep this method public and static.
-   */
-  public static ConfigCacheClient getCacheClient(
+  private ConfigCacheClient getCacheClient(
```

Comment:
from the javadoc seems like the only usefulness of the method was for use with the `LegacyConfigsHandler`, is it still needed?

Reply:
Ah yeah - don't need this method and the one above.